### ISSUE TITLE
LG webOS not found

### ISSUE THREAD

**Issue by sheinz** [benzea/gnome-screencast#75]
Hi

I can't connect to my LG TV version: 4.7.0-50706
Logs:
```
(gnome-network-displays:29935): GLib-GIO-DEBUG: 23:56:38.478: _g_io_module_get_default: Found default implementation dconf (DConfSettingsBackend) for ‘gsettings-backend’
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.478: watch_fast: "/org/gnome/desktop/interface/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/settings-daemon/peripherals/mouse/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/desktop/sound/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/desktop/privacy/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/desktop/wm/preferences/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/settings-daemon/plugins/xsettings/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.479: watch_fast: "/org/gnome/desktop/a11y/" (establishing: 0, active: 0)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.480: watch_established: "/org/gnome/desktop/interface/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.480: watch_established: "/org/gnome/settings-daemon/peripherals/mouse/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.481: watch_established: "/org/gnome/desktop/sound/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.481: watch_established: "/org/gnome/desktop/privacy/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.481: watch_established: "/org/gnome/desktop/wm/preferences/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.481: watch_established: "/org/gnome/settings-daemon/plugins/xsettings/" (establishing: 1)
(gnome-network-displays:29935): dconf-DEBUG: 23:56:38.481: watch_established: "/org/gnome/desktop/a11y/" (establishing: 1)
(gnome-network-displays:29935): GLib-GIO-DEBUG: 23:56:38.583: _g_io_module_get_default: Found default implementation gvfs (GDaemonVfs) for ‘gio-vfs’
(gnome-network-displays:29935): Gtk-DEBUG: 23:56:38.672: Connecting to session manager
** (gnome-network-displays:29935): DEBUG: 23:56:38.771: NdScreencastPortal: Aquired Portal proxy
** (gnome-network-displays:29935): DEBUG: 23:56:38.771: task is: 0x55fffe6dbd70
** (gnome-network-displays:29935): DEBUG: 23:56:38.772: NdPulseaudio: Querying sink info by name
** (gnome-network-displays:29935): DEBUG: 23:56:38.776: NdPulseaudio: Error querying sink info
** (gnome-network-displays:29935): DEBUG: 23:56:38.776: NdPulseaudio: Got a sink info for the expected name
** (gnome-network-displays:29935): DEBUG: 23:56:38.779: WFDP2PRegistry: Found a new device, creating provider
** (gnome-network-displays:29935): DEBUG: 23:56:38.779: WfdP2PProvider: Discover is now set to 1
** (gnome-network-displays:29935): DEBUG: 23:56:38.781: NdScreencastPortal: Received CreateSession response
** (gnome-network-displays:29935): DEBUG: 23:56:38.781: simple variant lookup: /org/freedesktop/portal/desktop/session/1_219/gtk1605188350
** (gnome-network-displays:29935): DEBUG: 23:56:38.791: WFDP2PRegistry: Got NMClient
** (gnome-network-displays:29935): DEBUG: 23:56:38.813: NdScreencastPortal: Received SelectSource response
** (gnome-network-displays:29935): DEBUG: 23:56:42.849: NdScreencastPortal: Received Start response
** (gnome-network-displays:29935): DEBUG: 23:56:42.849: Got a stream with node ID: 19
(gnome-network-displays:29935): dconf-DEBUG: 23:56:50.986: sync
```
My system is up to date Arch Linux.
I also can connect to the TV with [miraclecast](https://github.com/albfan/miraclecast)

I would appreciate any help how to diagnose the problem.

nmcli device show:
```
GENERAL.DEVICE:                         p2p-dev-wlp61s0
GENERAL.TYPE:                           wifi-p2p
GENERAL.HWADDR:                         (unknown)
GENERAL.MTU:                            0
GENERAL.STATE:                          30 (disconnected)
GENERAL.CONNECTION:                     --
GENERAL.CON-PATH:                       --
```

**Comment by benzea** [comment]
Are you able to quickly recompile things?

Commit d8baecb81a039d030d46e9f515a098fd7ba1ff2f might result in a bit more information. Other than that, you could check logs and dbus.

 1. journalctl should show something for wpa_supplicant and NetworkManager
 2. `gdbus introspect -y -d fi.w1.wpa_supplicant1 -o /fi/w1/wpa_supplicant1 -r` allows dumping wpa_supplicant objects
 3. e.g. d-feet allows browsing NetworkManager

In both cases, Peer objects should appear. Theoretically it could also be that the TV does not like our WFDIEs and simply never responds.

**Comment by sheinz** [comment]
Thanks for the reply. I will try that when I get home.

**Comment by odinho** [comment]
@sheinz did you try again? possibly due to the network-manager issue from the linked chain of issues?

**Comment by sheinz** [comment]
I've tried again with the latest changes and it kind of work.
My TV is reliably detected but connection is kind of glitchy.
I see a notification on the TV when I try to connect. After accepting, TV shows that connection is established, but nothing happens. 
After several attempts I manage to get streaming to work.

I have attached the logs in case it might be helpful to anyone.
[log.txt](https://github.com/benzea/gnome-network-displays/files/4283332/log.txt)

By the way, the TV firmware has also been updated, so it might also influenced the results.
